testbench/scripts/get-dishes.sh

```diff
@@ -10,6 +10,6 @@ if [ "#$ID_TOKEN" == "#" ] || [ "#$SESSION_TOKEN" == "#" ] ; then
 fi
 
 curl -i -X GET \
-    -H "x-api-key: test" \
+    -H "x-api-key: $API_KEY" \
```

Comment:
For local dev testing `x-api-key` will always be set to test by default. So can you please update it to use test?